### PR TITLE
Fix cmucl dump

### PR DESCRIPTION
**Pull request by guicho271828**
This is a first milestone to the tree shaker. #129

### CODE REVIEW

**Comment by guicho271828** [comment]
because the function is too long. cmucl-fix is just a side effect.

File: lisp/init.ros [hunk 1]
```diff
@@ -46,7 +46,10 @@ exec ros -Q -m roswell -L sbcl-bin -- $0 "$@"
                                 "#|-*- mode:lisp -*-|#"
                                 "#| <Put a one-line description here>"
                                 "exec ros -Q -- $0 \"$@\"" "|#"
-                                "(progn ;;init forms" "  #+quicklisp (ql:quickload '() :silent t))" ""
```
Comment (snmsts) [inline lisp/init.ros]: https://github.com/roswell/roswell/pull/181#pullrequestreview-3594360

Comment (guicho271828) [inline lisp/init.ros]: legit

**Comment by guicho271828** [comment]
btw also workImg on this to support ecl

**Comment by guicho271828** [comment]
oh, no, the code youve merged is still dirty and errors when you dump.
I didnt pushed the fix for your comment yet, too.

**Pull request merged**